gravitee-plugin-core/src/main/java/io/gravitee/plugin/core/internal/PluginRegistryImpl.java

```diff
@@ -93,10 +101,16 @@ public class PluginRegistryImpl extends AbstractService implements PluginRegistr
      */
     public PluginRegistryImpl() {}
 
-    public PluginRegistryImpl(String workspacePath) {
+    @VisibleForTesting
+    PluginRegistryImpl(String workspacePath) {
         this.workspacesPath = new String[] { workspacePath };
     }
 
+    @VisibleForTesting
```

Comment:
No needed I think, method is public.

Reply:
same as above, the visibility should be reduced

Reply:
I'll reduce visibility